services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/multitable/MultitableFlowsSpec.groovy

```diff
@@ -0,0 +1,1172 @@
+package org.openkilda.functionaltests.spec.multitable
+
+import static groovyx.gpars.GParsPool.withPool
+import static org.junit.Assume.assumeTrue
+import static org.openkilda.functionaltests.extension.tags.Tag.SMOKE
+import static org.openkilda.functionaltests.extension.tags.Tag.SMOKE_SWITCHES
+import static org.openkilda.testing.Constants.EGRESS_RULE_MULTI_TABLE_ID
+import static org.openkilda.testing.Constants.INGRESS_RULE_MULTI_TABLE_ID
+import static org.openkilda.testing.Constants.PATH_INSTALLATION_TIME
+import static org.openkilda.testing.Constants.RULES_INSTALLATION_TIME
+import static org.openkilda.testing.Constants.SINGLE_TABLE_ID
+import static org.openkilda.testing.Constants.TRANSIT_RULE_MULTI_TABLE_ID
+import static org.openkilda.testing.Constants.WAIT_OFFSET
+
+import org.openkilda.functionaltests.HealthCheckSpecification
+import org.openkilda.functionaltests.extension.tags.Tags
+import org.openkilda.functionaltests.helpers.PathHelper
+import org.openkilda.functionaltests.helpers.Wrappers
+import org.openkilda.messaging.command.switches.DeleteRulesAction
+import org.openkilda.messaging.error.MessageError
+import org.openkilda.messaging.info.event.IslChangeType
+import org.openkilda.messaging.info.event.PathNode
+import org.openkilda.messaging.payload.flow.FlowState
+import org.openkilda.model.Cookie
+import org.openkilda.model.SwitchFeature
+import org.openkilda.model.SwitchId
+import org.openkilda.northbound.dto.v1.flows.PingInput
+import org.openkilda.northbound.dto.v1.switches.SwitchPropertiesDto
+import org.openkilda.northbound.dto.v1.switches.SwitchSyncResult
+import org.openkilda.northbound.dto.v1.switches.SwitchValidationResult
+import org.openkilda.testing.model.topology.TopologyDefinition.Switch
+import org.openkilda.testing.service.traffexam.TraffExamService
+import org.openkilda.testing.tools.FlowTrafficExamBuilder
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.web.client.HttpClientErrorException
+import spock.lang.See
+
+import javax.inject.Provider
+
+@See("https://github.com/telstra/open-kilda/tree/develop/docs/design/multi-table-pipelines")
+class MultitableFlowsSpec extends HealthCheckSpecification {
+
+    @Autowired
+    Provider<TraffExamService> traffExamProvider
+
+    @Tags([SMOKE, SMOKE_SWITCHES])
+    def "System can use both single-table and multi-table switches in flow path at the same time, change switch table \
```

Comment:
I'd prefer to see here info that this case include protected flows

Reply:
my fault
I forgot to update the step description
fixed